src/RimDev.Stuntman.AspNetCore/common/StuntmanOptions.cs

```diff
@@ -83,6 +83,16 @@ public StuntmanOptions AddUser(StuntmanUser user)
             return AddUser(user, Constants.StuntmanOptions.LocalSource);
         }
 
+        public StuntmanOptions AddUsers(IEnumerable<StuntmanUser> users) 
```

Comment:
I started with `params` but the current version of C# doesn't support `params IEnumerable<T>` just yet.